src/relations/mysql.py

```diff
@@ -171,8 +194,8 @@ def _on_mysql_relation_created(self, event: RelationCreatedEvent) -> None:
         # Only execute handler if config values are set
         # else we'd be unable to create database and user
         if not username or not database:
-            logger.warning("mysql-interface-user or mysql-interface-database not set")
-            self.charm.unit.status = BlockedStatus("Missing `mysql` relation data")
+            logger.warning("Missing mysql-interface-user or mysql-interface-database")
+            self.charm.unit.status = BlockedStatus("Missing `mysql` relation configuration")
```

Comment:
This is something I received a feedback on. The next question is: What should I do?
What about to "hint" and not "inform": "Missing config mysql-interface-user/database"?

Reply:
[86c8749](https://github.com/canonical/mysql-k8s-operator/pull/214/commits/86c8749e84e196a5fc37ff79899a1eec6d101371)